rockylinux/README-short.txt

```diff
@@ -0,0 +1 @@
+The official build of Rocky Linux.
```

Comment:
This will show up in places like `docker search` (and https://hub.docker.com/search?type=image&image_filter=official), and will thus look something like this:

```console
$ docker search rockylinux
NAME        DESCRIPTION                         ...
rockylinux  The official build of Rocky Linux.  ...
```

This is fine if this is really what you want, but I would suggest that this is probably a good place for an "elevator pitch" for what Rocky Linux is.

Reply:
Discussed this with the team and we're OK with this language. It fits the upstream as well as other images e.g.